core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -116,7 +116,7 @@
  * This block master manages the metadata for all the blocks and block workers in Alluxio.
```

Comment:
can you specify the updated locking hierarchy here? like what we did in javadoc of `DefaultFileSystemMaster`

Reply:
Updated @apc999 Please check the section below. In `DefaultFileSystemMaster` it's the same pattern where these comments are not in the class commend, but somewhere in the class. I'm just following that w/o knowing why.

Reply:
See the big comment block below